package.json

```diff
@@ -25,5 +25,8 @@
     "eslint-plugin-promise": "^6.1.1",
     "prettier": "^3.2.5",
     "typescript": "^5.4.5"
+  },
+  "dependencies": {
+    "@changesets/cli": "^2.27.2"
```

Comment:
I wonder if changesets could go to devDependencies. I know in docs it says they should be in main, but that seems to be off (i.e. it is a tool not dependency of a project).

Reply:
Let's test it in the next PR thou as well.

Reply:
Cool, we can give it a try in next phase